assets/js/mobile-app-banner.js

```diff
@@ -11,7 +11,7 @@ const isNotCommuterRail = url => {
 };
 
 const isSchedulePage = url => {
-  return /\/schedules\/[\w.-]+\/line/.test(url);
+  return isNotCommuterRail(url) && /\/schedules\/[\w.-]+\/line/.test(url);
 };
```

Comment:
I don't see what is not commuter rail condition has anything to do with whether it's a schedule page, did we mix something up here

Reply:
Yeah, it's a little weird because they want it to be on schedule pages that aren't commuter rail schedules.

Reply:
So, yeah, there wouldn't have been overlap between the two. I thought it was better to communicate that it shouldn't be commuter rail even though they won't overlap. But, I can remove it.